include_dir_impl/src/dir.rs

```diff
@@ -1,15 +1,15 @@
 use failure::{self, Error, ResultExt};
 use file::File;
 use proc_macro2::TokenStream;
-use quote::ToTokens;
+use quote::{ToTokens, quote};
 use std::path::{Path, PathBuf};
 
 #[derive(Debug, Clone, PartialEq)]
-pub struct Dir {
-    pub root_rel_path: PathBuf,
-    pub abs_path: PathBuf,
-    pub files: Vec<File>,
-    pub dirs: Vec<Dir>,
+pub(crate) struct Dir {
```

Comment:
I doubt anyone except us would use `include_dir_impl` directly, but it's a good idea to keep things as private as possible to reduce inter-module coupling :+1: